packages/vuetify/src/labs/VInfiniteScroll/VInfiniteScroll.tsx

```diff
@@ -180,6 +190,13 @@ export const VInfiniteScroll = genericComponent<VInfiniteScrollSlots>()({
           if (status === 'ok' && side === 'start') {
             setScrollAmount(getScrollSize() - previousScrollSize + getScrollAmount())
           }
+          if (props.mode !== 'manual') {
+            window.requestAnimationFrame(() => {
+              window.requestAnimationFrame(() => {
```

Comment:
I can observe that quite often it loads twice on one scroll, presumably because of a race condition between when `isIntersecting` value is updated and `intersecting()` is called. If I added one more `requestAnimationFrame` it seemed a lot more stable, but I imagine it's not a very stable solution, and even three of them could break depending on browser and device 🤷

Reply:
Make sense. Leave it with me, I think I found something convincible and need more time prove.

Reply:
After further playing around, it is not caused by race condition between `isIntersecting` value and `intersecting()` is called.

The cause is browser takes 2 - 3 animation frames to trigger `IntersectionObserver` after `VInfiniteScrollIntersect` being pushed outside viewpoint. I think this is a browser behaviour, which means could be out of our control.

So far I couldn't come up with a better solution than using 3 nested `window.requestAnimationFrame`.

As per my testing, 3 nested `window.requestAnimationFrame` 100% works